test/functorch/test_control_flow.py

```diff
@@ -796,8 +801,8 @@ def forward(self, x_1, pred_1, pred2_1):
 
         code = graph.true_graph_0.code
         out = """
-        def forward(self, y_1):
-            mul = torch.ops.aten.mul.Tensor(y_1, y_1);  y_1 = None
+        def forward(self, arg0_1):
+            mul = torch.ops.aten.mul.Tensor(arg0_1, arg0_1);  arg0_1 = None
```

Comment:
(not blocking) I don't know why we're using self.assertEqual for these, they should all be self.assertExpectedInline

Reply:
Can clean them up in next diff.